buf/registry/owner/v1/organization.proto

```diff
@@ -72,6 +79,21 @@ enum OrganizationVerificationStatus {
   ORGANIZATION_VERIFICATION_STATUS_OFFICIAL = 3;
 }
 
+// An access role to a module.
+enum ModuleRole {
```

Comment:
Should this type belong in `buf.registry.module` instead of `buf.registry.organization`?